### PR TITLE
Fix webpack 5 warning + disabling of profiling

### PR DESCRIPTION
**Pull request by timneutkens**
Noticed that there was a specific warning when using webpack 5.

### CODE REVIEW

**Review by Timer — DISMISSED** [review]

**Review by Timer — DISMISSED** [review]

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| buildDuration | 8.7s | 8.6s | -19ms |
| nodeModulesSize | 77.6 MB | 77.6 MB | ⚠️ +1.48 kB |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 1.779 | 1.799 | ⚠️ +0.02 |
| / avg req/sec | 1404.96 | 1389.48 | ⚠️ -15.48 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.142 | 1.127 | -0.01 |
| /error-in-render avg req/sec | 2188.31 | 2219.11 | +30.8 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| 677f882d2ed8..43e3.js gzip | 13.1 kB | 13.1 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-9e3e4e6..2023.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-50be..df5b.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59.4 kB | 59.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| polyfills-af..9390.js gzip | 31.3 kB | 31.3 kB | ✓ |
| Overall change | 31.3 kB | 31.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| _app-2a09aa2..4a98.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-8b758..aef6.js gzip | 3.46 kB | 3.46 kB | ✓ |
| hooks-c71ae4..70cd.js gzip | 887 B | 887 B | ✓ |
| index-bbee2f..528b.js gzip | 227 B | 227 B | ✓ |
| link-7faf09b..eba4.js gzip | 1.64 kB | 1.64 kB | ✓ |
| routerDirect..bf84.js gzip | 303 B | 303 B | ✓ |
| withRouter-a..5826.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.09 kB | 8.09 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| Overall change | 321 B | 321 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| index.html gzip | 614 B | 614 B | ✓ |
| link.html gzip | 621 B | 621 B | ✓ |
| withRouter.html gzip | 608 B | 608 B | ✓ |
| Overall change | 1.84 kB | 1.84 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| buildDuration | 10.3s | 10.3s | -1ms |
| nodeModulesSize | 72.9 MB | 77.6 MB | ⚠️ +4.7 MB |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| 677f882d2ed8..43e3.js gzip | 13.1 kB | 13.1 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-9e3e4e6..2023.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-50be..df5b.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59.4 kB | 59.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| polyfills-af..9390.js gzip | 31.3 kB | 31.3 kB | ✓ |
| Overall change | 31.3 kB | 31.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| _app-2a09aa2..4a98.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-8b758..aef6.js gzip | 3.46 kB | 3.46 kB | ✓ |
| hooks-c71ae4..70cd.js gzip | 887 B | 887 B | ✓ |
| index-bbee2f..528b.js gzip | 227 B | 227 B | ✓ |
| link-7faf09b..eba4.js gzip | 1.64 kB | 1.64 kB | ✓ |
| routerDirect..bf84.js gzip | 303 B | 303 B | ✓ |
| withRouter-a..5826.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.09 kB | 8.09 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| Overall change | 321 B | 321 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| _error.js | 1 MB | 1 MB | ✓ |
| 404.html | 2.67 kB | 2.67 kB | ✓ |
| hooks.html | 1.92 kB | 1.92 kB | ✓ |
| index.js | 1 MB | 1 MB | ✓ |
| link.js | 1.06 MB | 1.06 MB | ✓ |
| routerDirect.js | 1.05 MB | 1.05 MB | ✓ |
| withRouter.js | 1.05 MB | 1.05 MB | ✓ |
| Overall change | 5.18 MB | 5.18 MB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Webpack 5 Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| buildDuration | 10.7s | 10.9s | ⚠️ +193ms |
| nodeModulesSize | 77.6 MB | 77.6 MB | ⚠️ +1.48 kB |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 1.756 | 1.726 | -0.03 |
| / avg req/sec | 1423.4 | 1448.56 | +25.16 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.105 | 1.093 | -0.01 |
| /error-in-render avg req/sec | 2263.33 | 2286.36 | +23.03 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| 597-3bd7d6bd..868b.js gzip | 13 kB | 13 kB | ✓ |
| 778-d412f8e7..3ee1.js gzip | 6.56 kB | 6.56 kB | ✓ |
| 838-adcbad39..6e83.js gzip | 31.1 kB | 31.1 kB | ✓ |
| framework.HASH.js gzip | 39.3 kB | 39.3 kB | ✓ |
| main-6a6a792..6877.js gzip | 96 B | 96 B | ✓ |
| webpack-e302..54e9.js gzip | 954 B | 954 B | ✓ |
| Overall change | 91 kB | 91 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| polyfills-a0..538c.js gzip | 88 B | 88 B | ✓ |
| Overall change | 88 B | 88 B | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| _app-a5b2e84..33cf.js gzip | 1.26 kB | 1.26 kB | ✓ |
| _error-6f1f9..5577.js gzip | 3.38 kB | 3.38 kB | ✓ |
| hooks-725de8..0a15.js gzip | 904 B | 904 B | ✓ |
| index-939503..6e1c.js gzip | 232 B | 232 B | ✓ |
| link-e0cc871..fdbb.js gzip | 1.63 kB | 1.63 kB | ✓ |
| routerDirect..9360.js gzip | 308 B | 308 B | ✓ |
| withRouter-6..44ec.js gzip | 304 B | 304 B | ✓ |
| Overall change | 8.02 kB | 8.02 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| _buildManifest.js gzip | 299 B | 299 B | ✓ |
| Overall change | 299 B | 299 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| index.html gzip | 585 B | 585 B | ✓ |
| link.html gzip | 592 B | 592 B | ✓ |
| withRouter.html gzip | 581 B | 581 B | ✓ |
| Overall change | 1.76 kB | 1.76 kB | ✓ |

</details>

#### Diffs
<details>
<summary>Diff for <strong>index.html</strong></summary>

```diff
@@ -43,7 +43,7 @@
         "props": { "pageProps": {} },
         "page": "/",
         "query": {},
-        "buildId": "e-8NuL7SeKv8srg9bH31x",
+        "buildId": "VLc-zFBNz-o6hzjRWoMpq",
         "isFallback": false,
         "gip": true
       }
@@ -77,11 +77,11 @@
       async=""
     ></script>
     <script
-      src="/_next/static/e-8NuL7SeKv8srg9bH31x/_buildManifest.js"
+      src="/_next/static/VLc-zFBNz-o6hzjRWoMpq/_buildManifest.js"
       async=""
     ></script>
     <script
-      src="/_next/static/e-8NuL7SeKv8srg9bH31x/_ssgManifest.js"
+      src="/_next/static/VLc-zFBNz-o6hzjRWoMpq/_ssgManifest.js"
       async=""
     ></script>
   </body>
```
</details>
<details>
<summary>Diff for <strong>link.html</strong></summary>

```diff
@@ -48,7 +48,7 @@
         "props": { "pageProps": {} },
         "page": "/link",
         "query": {},
-        "buildId": "e-8NuL7SeKv8srg9bH31x",
+        "buildId": "VLc-zFBNz-o6hzjRWoMpq",
         "isFallback": false,
         "gip": true
       }
@@ -82,11 +82,11 @@
       async=""
     ></script>
     <script
-      src="/_next/static/e-8NuL7SeKv8srg9bH31x/_buildManifest.js"
+      src="/_next/static/VLc-zFBNz-o6hzjRWoMpq/_buildManifest.js"
       async=""
     ></script>
     <script
-      src="/_next/static/e-8NuL7SeKv8srg9bH31x/_ssgManifest.js"
+      src="/_next/static/VLc-zFBNz-o6hzjRWoMpq/_ssgManifest.js"
       async=""
     ></script>
   </body>
```
</details>
<details>
<summary>Diff for <strong>withRouter.html</strong></summary>

```diff
@@ -43,7 +43,7 @@
         "props": { "pageProps": {} },
         "page": "/withRouter",
         "query": {},
-        "buildId": "e-8NuL7SeKv8srg9bH31x",
+        "buildId": "VLc-zFBNz-o6hzjRWoMpq",
         "isFallback": false,
         "gip": true
       }
@@ -77,11 +77,11 @@
       async=""
     ></script>
     <script
-      src="/_next/static/e-8NuL7SeKv8srg9bH31x/_buildManifest.js"
+      src="/_next/static/VLc-zFBNz-o6hzjRWoMpq/_buildManifest.js"
       async=""
     ></script>
     <script
-      src="/_next/static/e-8NuL7SeKv8srg9bH31x/_ssgManifest.js"
+      src="/_next/static/VLc-zFBNz-o6hzjRWoMpq/_ssgManifest.js"
       async=""
     ></script>
   </body>
```
</details>
</details>
Commit: 8758137758ee4e9d8c54d6a235c2c9e04c3e907d

**Review by Timer — APPROVED** [review]

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| buildDuration | 10.6s | 10.7s | ⚠️ +160ms |
| nodeModulesSize | 77.6 MB | 77.6 MB | ⚠️ +1.48 kB |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.132 | 2.156 | ⚠️ +0.02 |
| / avg req/sec | 1172.56 | 1159.51 | ⚠️ -13.05 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.343 | 1.36 | ⚠️ +0.02 |
| /error-in-render avg req/sec | 1862.1 | 1838.83 | ⚠️ -23.27 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| 677f882d2ed8..43e3.js gzip | 13.1 kB | 13.1 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-9e3e4e6..2023.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-50be..df5b.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59.4 kB | 59.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| polyfills-af..9390.js gzip | 31.3 kB | 31.3 kB | ✓ |
| Overall change | 31.3 kB | 31.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| _app-2a09aa2..4a98.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-8b758..aef6.js gzip | 3.46 kB | 3.46 kB | ✓ |
| hooks-c71ae4..70cd.js gzip | 887 B | 887 B | ✓ |
| index-bbee2f..528b.js gzip | 227 B | 227 B | ✓ |
| link-7faf09b..eba4.js gzip | 1.64 kB | 1.64 kB | ✓ |
| routerDirect..bf84.js gzip | 303 B | 303 B | ✓ |
| withRouter-a..5826.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.09 kB | 8.09 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| Overall change | 321 B | 321 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| index.html gzip | 614 B | 614 B | ✓ |
| link.html gzip | 621 B | 621 B | ✓ |
| withRouter.html gzip | 608 B | 608 B | ✓ |
| Overall change | 1.84 kB | 1.84 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| buildDuration | 12.9s | 12.4s | -501ms |
| nodeModulesSize | 77.6 MB | 77.6 MB | ⚠️ +1.48 kB |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| 677f882d2ed8..43e3.js gzip | 13.1 kB | 13.1 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-9e3e4e6..2023.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-50be..df5b.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59.4 kB | 59.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| polyfills-af..9390.js gzip | 31.3 kB | 31.3 kB | ✓ |
| Overall change | 31.3 kB | 31.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| _app-2a09aa2..4a98.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-8b758..aef6.js gzip | 3.46 kB | 3.46 kB | ✓ |
| hooks-c71ae4..70cd.js gzip | 887 B | 887 B | ✓ |
| index-bbee2f..528b.js gzip | 227 B | 227 B | ✓ |
| link-7faf09b..eba4.js gzip | 1.64 kB | 1.64 kB | ✓ |
| routerDirect..bf84.js gzip | 303 B | 303 B | ✓ |
| withRouter-a..5826.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.09 kB | 8.09 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| Overall change | 321 B | 321 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| _error.js | 1 MB | 1 MB | ✓ |
| 404.html | 2.67 kB | 2.67 kB | ✓ |
| hooks.html | 1.92 kB | 1.92 kB | ✓ |
| index.js | 1 MB | 1 MB | ✓ |
| link.js | 1.06 MB | 1.06 MB | ✓ |
| routerDirect.js | 1.05 MB | 1.05 MB | ✓ |
| withRouter.js | 1.05 MB | 1.05 MB | ✓ |
| Overall change | 5.18 MB | 5.18 MB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Webpack 5 Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| buildDuration | 13.5s | 13.5s | -24ms |
| nodeModulesSize | 77.6 MB | 77.6 MB | ⚠️ +1.48 kB |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.073 | 2.041 | -0.03 |
| / avg req/sec | 1205.76 | 1225.05 | +19.29 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.36 | 1.362 | 0 |
| /error-in-render avg req/sec | 1838.48 | 1836.12 | ⚠️ -2.36 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| 597-3bd7d6bd..868b.js gzip | 13 kB | 13 kB | ✓ |
| 778-d412f8e7..3ee1.js gzip | 6.56 kB | 6.56 kB | ✓ |
| 838-adcbad39..6e83.js gzip | 31.1 kB | 31.1 kB | ✓ |
| framework.HASH.js gzip | 39.3 kB | 39.3 kB | ✓ |
| main-6a6a792..6877.js gzip | 96 B | 96 B | ✓ |
| webpack-e302..54e9.js gzip | 954 B | 954 B | ✓ |
| Overall change | 91 kB | 91 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| polyfills-a0..538c.js gzip | 88 B | 88 B | ✓ |
| Overall change | 88 B | 88 B | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| _app-a5b2e84..33cf.js gzip | 1.26 kB | 1.26 kB | ✓ |
| _error-6f1f9..5577.js gzip | 3.38 kB | 3.38 kB | ✓ |
| hooks-725de8..0a15.js gzip | 904 B | 904 B | ✓ |
| index-939503..6e1c.js gzip | 232 B | 232 B | ✓ |
| link-e0cc871..fdbb.js gzip | 1.63 kB | 1.63 kB | ✓ |
| routerDirect..9360.js gzip | 308 B | 308 B | ✓ |
| withRouter-6..44ec.js gzip | 304 B | 304 B | ✓ |
| Overall change | 8.02 kB | 8.02 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| _buildManifest.js gzip | 299 B | 299 B | ✓ |
| Overall change | 299 B | 299 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js fix/webpack5-deprecation-warning-profiling | Change |
| - | - | - | - |
| index.html gzip | 585 B | 585 B | ✓ |
| link.html gzip | 592 B | 592 B | ✓ |
| withRouter.html gzip | 581 B | 581 B | ✓ |
| Overall change | 1.76 kB | 1.76 kB | ✓ |

</details>

#### Diffs
<details>
<summary>Diff for <strong>index.html</strong></summary>

```diff
@@ -43,7 +43,7 @@
         "props": { "pageProps": {} },
         "page": "/",
         "query": {},
-        "buildId": "oQQB4S5Q4cpdqXtdHcjs7",
+        "buildId": "gJKunmmr10JrwNeKYxyE2",
         "isFallback": false,
         "gip": true
       }
@@ -77,11 +77,11 @@
       async=""
     ></script>
     <script
-      src="/_next/static/oQQB4S5Q4cpdqXtdHcjs7/_buildManifest.js"
+      src="/_next/static/gJKunmmr10JrwNeKYxyE2/_buildManifest.js"
       async=""
     ></script>
     <script
-      src="/_next/static/oQQB4S5Q4cpdqXtdHcjs7/_ssgManifest.js"
+      src="/_next/static/gJKunmmr10JrwNeKYxyE2/_ssgManifest.js"
       async=""
     ></script>
   </body>
```
</details>
<details>
<summary>Diff for <strong>link.html</strong></summary>

```diff
@@ -48,7 +48,7 @@
         "props": { "pageProps": {} },
         "page": "/link",
         "query": {},
-        "buildId": "oQQB4S5Q4cpdqXtdHcjs7",
+        "buildId": "gJKunmmr10JrwNeKYxyE2",
         "isFallback": false,
         "gip": true
       }
@@ -82,11 +82,11 @@
       async=""
     ></script>
     <script
-      src="/_next/static/oQQB4S5Q4cpdqXtdHcjs7/_buildManifest.js"
+      src="/_next/static/gJKunmmr10JrwNeKYxyE2/_buildManifest.js"
       async=""
     ></script>
     <script
-      src="/_next/static/oQQB4S5Q4cpdqXtdHcjs7/_ssgManifest.js"
+      src="/_next/static/gJKunmmr10JrwNeKYxyE2/_ssgManifest.js"
       async=""
     ></script>
   </body>
```
</details>
<details>
<summary>Diff for <strong>withRouter.html</strong></summary>

```diff
@@ -43,7 +43,7 @@
         "props": { "pageProps": {} },
         "page": "/withRouter",
         "query": {},
-        "buildId": "oQQB4S5Q4cpdqXtdHcjs7",
+        "buildId": "gJKunmmr10JrwNeKYxyE2",
         "isFallback": false,
         "gip": true
       }
@@ -77,11 +77,11 @@
       async=""
     ></script>
     <script
-      src="/_next/static/oQQB4S5Q4cpdqXtdHcjs7/_buildManifest.js"
+      src="/_next/static/gJKunmmr10JrwNeKYxyE2/_buildManifest.js"
       async=""
     ></script>
     <script
-      src="/_next/static/oQQB4S5Q4cpdqXtdHcjs7/_ssgManifest.js"
+      src="/_next/static/gJKunmmr10JrwNeKYxyE2/_ssgManifest.js"
       async=""
     ></script>
   </body>
```
</details>
</details>
Commit: c1ac6c78535414cc975c8c8b1743c8e67c870c61

File: packages/next/build/webpack/plugins/profiling-plugin.ts [hunk 1]
```diff
@@ -1,13 +1,28 @@
...
+
 export class ProfilingPlugin {
   apply(compiler: any) {
     // Only enabled when instrumentation is loaded
-    if (!tracer.getCurrentSpan()) {
+    const currentSpan = tracer.getCurrentSpan()
+    console.log({
+      currentSpan,
+      isRecording: currentSpan?.isRecording(),
+    })
```
Comment (kaykdm) [inline packages/next/build/webpack/plugins/profiling-plugin.ts:24]: @timneutkens 
Is this `console.log` kept intentionally?

Comment (timneutkens) [reply packages/next/build/webpack/plugins/profiling-plugin.ts:24]: It wasn't, totally missed this!

**Pull request merged**